test/e2e/upgrade/upgrade.go

```diff
@@ -404,6 +404,11 @@ func clusterUpgrade(f *framework.Framework, c configv1client.Interface, dc dynam
 			return nil, false
 		},
 	); err != nil {
+		//before returning the err force a failure for completes upgrade which follows this test
+		disruption.RecordJUnit(f, "[sig-cluster-lifecycle] Cluster completes upgrade", func() (error, bool) {
+			framework.Logf("Cluster version operator failed to acknowledge upgrade request")
+			return fmt.Errorf("Cluster did not complete %s: %s", "upgrade", "operator failed to acknowledge upgrade request"), false
```

Comment:
No need for the formatting here just a static string should do the trick.

Reply:
Went back and forth as errors.New wasn't imported and calling fmt without formatting felt odd too.  Will go with fmt.Errorf with a constant string for now, but can also pull in errrors if preferred.